aws/resource_aws_egress_only_internet_gateway.go

```diff
@@ -82,6 +82,19 @@ func resourceAwsEgressOnlyInternetGatewayRead(d *schema.ResourceData, meta inter
 	return nil
 }
 
+func gatewayFound(id string, resp *ec2.DescribeEgressOnlyInternetGatewaysOutput) bool {
```

Comment:
Nit: Until the project is split up into service-based Go packages, we should ensure our naming makes sense across all AWS functionality -- so in this case adding that its for EC2, e.g.

```suggestion
func hasEc2EgressOnlyInternetGateway(id string, resp *ec2.DescribeEgressOnlyInternetGatewaysOutput) bool {
```